app/(pages)/evidence/page.tsx

```diff
@@ -1,7 +1,22 @@
 import { FC } from 'react';
 
+import "./evidence.scss"
+import Form from '@/app/components/Evidence/form';
+import SideImage from '@/app/components/Evidence/sideImage';
+
+
+
 const Evidence: FC = () => {
-  return <div>Evidence</div>;
+  return  <div className="evidence__container"> 
+  <div className="sm:flex w-full"> 
+    <div className="lg:w-1/2 lg:pt-10 sm:w-full  ">
+      <Form />
```

Comment:
Please use another name for this component as there is already and form tag in html.